src/transport/SecureSessionMgr.cpp

```diff
@@ -92,136 +92,157 @@ CHIP_ERROR SecureSessionMgr::SendMessage(NodeId peerNodeId, System::PacketBuffer
 CHIP_ERROR SecureSessionMgr::SendMessage(PayloadHeader & payloadHeader, NodeId peerNodeId, System::PacketBufferHandle msgBuf,
```

Comment:
Should we call these ResendMessage?

I think there should be a difference between stuff taking headers and things that decode packet/payload headers.

Reply:
How about `SendEncryptedMessage`? From the API perspective, it does not know the CRMP is using it for resends.